lib/hterm.js

```diff
@@ -1,16 +1,42 @@
 import {hterm, lib} from 'hterm-umdjs';
+import unicodeStringUtils from 'unicode-string-utils';
```

Comment:
I wanna propose [runes](https://github.com/dotcypress/runes) library for splitting unicode strings. Because `unicode-string-utils` don't support some emoji subset.(👩‍👩‍👧‍👦,❤️,👍🏽 etc)